packages/portalnetwork/src/wire/utp/Protocol/write/ContentWriter.ts

```diff
@@ -62,18 +63,18 @@ export default class ContentWriter {
     this.logger('All Data Written')
   }
 
-  chunk(content: Uint8Array, size: number): Record<number, Uint8Array> {
-    let arrayMod = content
+  chunk(): Record<number, Uint8Array> {
+    let arrayMod = this.content
     this.logger(`Preparing`)
-    this.logger(content)
-    this.logger(`For transfer as ${size} byte chunks.`)
-    const full = Math.floor(content.length / size)
-    const partial = content.length % size > 0 ? 1 : 0
+    this.logger(this.content)
```

Comment:
Is there a reason for us to dump the entire bytestring to the logs? Maybe we could log just the first 20 bytes or something if it has some use but otherwise just feels overly verbose.

Reply:
No, not at all.  Just a relic from me working on it. I just like seeing each thing at every stage while I build.  I'm starting to understand how writing tests as I go accomplishes the same thing and more, but *i'm just a baby*